lib/screens/home/home_screen.dart

```diff
@@ -22,13 +22,24 @@ class _HomeScreenState extends State<HomeScreen> {
   CalendarController _calendarController = CalendarController();
   final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
   Stream<List<Wod>> _wodsStream;
+  Map<DateTime, List<Wod>> _data;
 
   @override
   void initState() {
     super.initState();
     _wodsStream = context.read<WodRepository>().getWods();
```

Comment:
Now that you have to implement the Calendar feature, you should ONLY fetch wods for that day from Firestore or you may have performance issues.

Imagine that your application would have been running for 10 years and you have at least 5 Workouts for each day.
With that code, you are fetching ALL Wods (like 18000 Wods) and then grouping them just to, in the end, show only 5 Wods.
This is might be A LOT of process for someone who doesn't have a top of the range cellphone.

What you really should do is just to fetch the workouts that have the selected day. Then, Firebase would only give you those 5 and you wouldn't even need to group them. Makes sense? :)